packages/components/tree-v2/src/defaults.ts

```diff
@@ -0,0 +1,117 @@
+import type { PropType } from 'vue'
+import type {
+  TreeNode,
+  TreeKey,
+  TreeOptionProps,
+  FilterMethod,
+} from './tree.type'
+
+export const treeProps = {
+  data: {
+    type: Array,
+    default: () => [],
+  },
+  emptyText: {
+    type: String,
+  },
+  height: {
+    type: Number,
+    default: 200,
+  },
+  props: {
+    type: Object as PropType<TreeOptionProps>,
+    default: () => ({
+      children: 'children',
+      label: 'label',
+      disabled: 'disabled',
+      value: 'id',
+    }),
+  },
```

Comment:
Can we use `buildProp`?

```ts
import { buildProps } from '@element-plus/utils/props`

props: buildProps({
 // params.
})

```

Reply:
I think we can replace all `PropType` to buildProp

Reply:
BTY, I think the filename should be `tree.ts`. We should unify the naming rules of components.
`kebab-case-name.vue` for Vue SFC, and `kebab-case-name.ts` for exporting prop and emit types.

Please reference `packages/components/affix/src`.